frontend/src/components/PReadsNavbar.js

```diff
@@ -78,6 +78,11 @@ export default class PReadsNavbar extends React.Component {
                   </NavItem>
                 );
               })}
+              {this.isLoggedIn() && (
```

Comment:
think this got added accidentally in the merge :curly_loop: 
log out should just be in the hamburger :hamburger: menu